tests/test_loader_deprecated.py

```diff
@@ -0,0 +1,695 @@
+"""
```

Comment:
Given `ItemLoader` is no longer deprecated, it may make sense to restore the original file name.

Reply:
Maybe a better name, these are the tests that were moved to the new library.
The idea is to remove them once the deprecated stuff (processors, ...) is removed.
`test_loader_moved_itemloaders.py`?

Reply:
Or maybe we can leave the `test_loader_deprecated` name but add a `test_loader` with the tests covering Scrapy-specific features that will remain after those deprecations.

Reply:
@Gallaecio , but there's already a file named `test_loader.py`, which is the past one, but I removed the tests that were covered by `itemloaders` (moved them to `test_loader_deprecated`)

Reply:
:facepalm: 

My bad, it did not load by default in the patch viewer and because it only had deleted lines in the count of added and deleted lines I though the whole file had been deleted.